client/src/main/java/org/spine3/base/Stringifier.java

```diff
@@ -20,12 +20,14 @@
 
 package org.spine3.base;
 
-import com.google.common.base.Function;
+import com.google.common.base.Converter;
 
 /**
- * An object converting to {@code String}.
+ * Converts an object of the I class to the {@code String} and reverse.
```

Comment:
`{@code I}`?

Also, let's reprhase `reverse` as it does not fit the sentence. Check the Guava's documentation on the same matter.